src/components/Navigation.spec.js

```diff
@@ -13,9 +13,11 @@ describe('Navigation', () => {
     const navigation = screen.getByRole('navigation');
     const seriesLink = screen.getByRole('link', { name: /serien/i });
     const movieLink = screen.getByRole('link', { name: /filme/i });
+    const searchLink = screen.getByRole('link', { name: /suche/i });
 
     expect(navigation).toBeInTheDocument();
     expect(seriesLink).toBeInTheDocument();
     expect(movieLink).toBeInTheDocument();
+    expect(searchLink).toBeInTheDocument();
```

Comment:
Unfortunately, I cannot comment above, but in your "it"-statement it says that two links will be rendered, but now it should be three. As well, you could search for all three links in one by using `screen.getAllByRole('link)`

Reply:
Thanks for the hint :)